packages/paste-website/src/components/shortcodes/feedback-popover/index.tsx

```diff
@@ -13,42 +13,45 @@ import {ThumbsDownIcon} from '@twilio-paste/icons/esm/ThumbsDownIcon';
 import {SupportIcon} from '@twilio-paste/icons/esm/SupportIcon';
 import {trackCustomEvent} from 'gatsby-plugin-google-analytics';
 import {SimpleStorage} from '../../../utils/SimpleStorage';
-import {getCurrentPathname} from '../../../utils/RouteUtils';
+import {useLocationPathname} from '../../../utils/RouteUtils';
```

Comment:
suggestion: This component directly interfacing with the route utils is causing complexities when it comes to trying to render in isolation (i.e. in Storybook). Would it make more sense to pass in the current location from a prop on the component call?